backup.sh

```diff
@@ -34,4 +34,7 @@ do
   fi
 done
 
-echo "Backup process finished at $(date "+%Y-%m-%d %H:%M:%S")"
\ No newline at end of file
+find $BACKUPDIR -type f -name '*.bak' -mtime +"$AGE_BACKUPS" -exec rm {} \;
```

Comment:
I am extra cautious here and avoid something like finding '*'. You never know what else writes to that directory in the future, better to be explicit.